Automattic/wp-calypso
# A4A: Extract/decouple A4A/Jetpack styles from SitesDashboard

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/307

## Proposed Changes

* This PR decouples some styles from sites-dashboard/style.scss files into an a4a only version - a4a-style.scss, where they would be relevant to A4A (or Jetpack Cloud) only.

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* To test, make sure that dataviews styles have not been impact at all - so making sure the dataviews table, columns, rows, pagination, filter and search display, all behave as expected.
* Check the stylesheet changes to see that they make sense and include all the would be expected.


## Pre-merge Checklist
